src/EventStore.Core/Services/RequestManager/RequestManagementService.cs

```diff
@@ -52,20 +53,22 @@ public class RequestManagementService :
 
 			_prepareTimeout = prepareTimeout;
 			_commitTimeout = commitTimeout;
+			_writeTimeout = writeTimeout;
 			_commitSource = new CommitSource();
 		}
 		
 		public void Handle(ClientMessage.WriteEvents message) {
 			var manager = new WriteEvents(
 								_bus,
-								_commitTimeout,
+								_writeTimeout,
```

Comment:
```suggestion
								_commitTimeout,
```
WriteTimeout should not need to be propagated further than where the cancellation token was created in each of the transports.